_includes/content/non-helm-manifests/install-calico.ps1

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2020 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+<#
+.DESCRIPTION
+    This script installs and starts Calico services on a Windows node.
+#>
+
+Param(
+    [parameter(Mandatory = $false)] $ReleaseBaseURL="https://github.com/projectcalico/node/releases/download/{{site.data.versions.first.title}}/",
+    [parameter(Mandatory = $false)] $ReleaseFile="calico-os-{{site.data.versions.first.title}}.zip",
```

Comment:
For this to work, we'll need to create a release named "master in the node repository.

Reply:
That isn't a great solution and I'd really rather not go down that path. What are our other options?

Reply:
Maybe a Google Cloud Storage bucket where the specific calico windows zip files are made public? I added an example GCS bucket name (tigera-windows). WDYT?

Reply:
We chatted about this online. Release versions will download the archive from the Calico release. Otherwise, the archive will be served from `files/`